common/beerocks/bcl/source/beerocks_ucc_listener.cpp

```diff
@@ -438,7 +438,13 @@ bool beerocks_ucc_listener::custom_message_handler(Socket *sd, uint8_t *rx_buffe
                                                    size_t rx_buffer_size)
 {
     std::string command_string;
-    auto available_bytes       = sd->readBytes(rx_buffer, rx_buffer_size, true);
+    auto available_bytes = sd->readBytes(rx_buffer, rx_buffer_size - 1, true);
+
+    if (available_bytes <= 0) {
+        LOG(ERROR) << "Cannot read from socket";
+        return true;
+    }
+
```

Comment:
Actually, if available_bytes is equal to the buffer size, it is also an error because the command will be incomplete and we're not going to read a second time and append to the buffer. But I guess handle_wfa_command will deal with that.